war/src/main/webapp/help/system-config/master-slave/description.html

```diff
@@ -1,11 +1,6 @@
 <div>
-  Optional human-readable description of this slave. This information
-  is displayed on the project configuration screen.
-
+  Optional human-readable description for this agent.
```

Comment:
Not sure. It doesn't really make a difference to the UI — I have a few agents with longer descriptions like "Mac Mini, OS X 10.11, 4-core, 32GB, Düsseldorf Datacentre" and nothing breaks :)